Formula/gdal.rb

```diff
@@ -83,16 +82,17 @@ def install
       "--with-proj=#{Formula["proj"].opt_prefix}",
       "--with-zstd=#{Formula["zstd"].opt_prefix}",
       "--with-liblzma=yes",
-      "--with-cfitsio=/usr/local",
-      "--with-hdf5=/usr/local",
-      "--with-netcdf=/usr/local",
-      "--with-jasper=/usr/local",
-      "--with-xerces=/usr/local",
-      "--with-odbc=/usr/local",
-      "--with-dods-root=/usr/local",
-      "--with-epsilon=/usr/local",
-      "--with-webp=/usr/local",
-      "--with-podofo=/usr/local",
+      "--with-cfitsio=#{Formula["proj"].opt_prefix}",
```

Comment:
Why is this `proj` and not `cfitsio`?

Reply:
Oops - thanks for spotting this. It's a typo; fixed in the next commit.